.github/workflows/tests.yml

```diff
@@ -0,0 +1,61 @@
+name: Run tests
+
+on:
+  push:
+    branches:
+      - 'test'
+  pull_request:
+    branches:
+      - 'main'
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.11
+
+      - name: Set secrets in .env file
+        run: |
+          cd ./src
```

Comment:
I suggest giving the variables in `secrets` exactly the same names as the variables we expect in `.env`.  That will help us avoid typos and confusion.

For example, `secrets. SLACK_SIGNING_SECRET ` instead of `secrets.SLACKSIGNINGSECRET `